workspaces/cli-server/src/server.ts

```diff
@@ -159,7 +159,7 @@ class CliServer {
       bodyParser.json({ limit: '5kb' }),
       async (req, res: express.Response) => {
         const { path, taskConfig, captureId } = req.body;
-        if (captureId && taskConfig) {
+        if (captureId) {
```

Comment:
I'm not sure why taskConfig was required for this - this creates the initial file with metadata